script.ebooks/resources/lib/kindleunpack/mobi_header.py

```diff
@@ -135,7 +169,7 @@ def dump_contexth(cpage, extheader):
         content = extheader[pos + 8: pos + size]
         if id in id_map_strings:
             name = id_map_strings[id]
-            print('\n    Key: "%s"\n        Value: "%s"' % (name, content.decode(codec)))
```

Comment:
This embedded lib is a free tool used for more than just Kodi, so it would be hard to change all the prints to logs as it would make updating to resolve bug fixes very hard.